analyzers/tests/SonarAnalyzer.UnitTest/SymbolicExecution/Roslyn/RoslynSymbolicExecutionTest.Invocation.cs

```diff
@@ -523,6 +522,47 @@ public void Invocation_Linq_VB()
             validator.ValidateTag("Value", x => x.Should().BeNull());
         }
 
+        [DataTestMethod]
+        [DataRow("Object = Nothing", true)]
+        [DataRow("Object = New Object()", false)]
+        [DataRow("Integer = Nothing", false)]   // While it can be assigned Nothing, value 0 is stored. And that is not null
```

Comment:
```suggestion
        [DataRow("Integer = Nothing", false)]   // While it can be assigned Nothing, value 0 is stored. And that is not null
        [DataRow("Integer? = Nothing", true)]
        [DataRow("TClass = Nothing", true)]
        [DataRow("TStruct = Nothing", false)]
        [DataRow("T = Nothing", true)]
```

Reply:
I've added those case, but the expectations are different